UrbanAirship.Android.Preference.nuspec

```diff
@@ -11,6 +11,7 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid">
+            <dependency id="Xamarin.AndroidX.Preference" version="1.0.0"/>
```

Comment:
I think version 1.1.0 is available.

Reply:
it is, but this is the version urbanairship-preference currently targets.